appveyor-windows-build-python.yml

```diff
@@ -38,6 +38,7 @@ install:
 
     # Make sure the temp directory exists for Python to use.
     # Install python3.8
+    - "choco install chocolatey-core.extension --version 1.3.3 --force -y"
```

Comment:
Why do we have to pass a version at all? Can't we always install the latest version? Or it don't work with later versions?

Reply:
https://ci.appveyor.com/project/AWSSAMCLI/aws-sam-cli-q551u/builds/29367692

From what we are installing we need 1.3.3, otherwise they fail. This has to do with what the package (or whatever choco calls them). Something got updated underneath us (not entirely sure what), so this is to get us back to a good state